scripts/gen-jsonschema.mts

```diff
@@ -4,6 +4,7 @@
 import fs from 'fs';
 import path from 'path';
 
+import { ConfigJsonSchema } from '../apps/prairielearn/src/lib/config.js';
```

Comment:
The problem here is that this config only gets the properties used in apps/prairielearn, not in grader-host or workspace-host. Note that `config.json` can serve all three.

Reply:
I see, I'll adjust for that. Thanks!

Reply:
Also, just for completeness:
* `config.json` in the root of PL sets options for all three apps;
* `apps/YYY/config.json` sets options for the `YYY` app.

So users have the option of having separate config for each app or a single file with all config.

Reply:
I did this in [a1b11f7](https://github.com/PrairieLearn/PrairieLearn/pull/12183/commits/a1b11f7da8409a7f8def404a4a69d33db2c72557).